qa/tests/features/new/manual/decision-tree/tree/tree.feature

```diff
@@ -0,0 +1,263 @@
+Feature: Decision Tree Panel, Decision Trees list
+	As the Anfisa user I want to see the Decision Trees list on the "Decision Tree Panel" page
+
+Background:
+	Given user opens "Decision Tree Panel" for the "PGP3140_wgs_panel_hl" dataset
+
+@regression
+Scenario Outline: 01 Search by valid "<ValidNameField>" value
+	When user clicks "Select Decision Tree"
+	And clicks "@BGM Research" decision tree
+	And clicks "Apply Filter" button
+	And inputs "<ValidNameField>" in algorithm search field
+	Then the field should be filtered
+
+Examples:
+	| ValidNameField |
+	| Proband_GQ     |
+	| Region_Worst   |
+
+@regression
+Scenario Outline: 02 Search by invalid value
+	Given "@BGM Research" was loaded
+	When user inputs "<InvalidNameField>" in algorithm search field
+	Then nothing should be shown
+
+Examples:
+	| ValidNameField |
+	| Anfisa         |
+
+Scenario Outline: 03 Search by "<PartValidNameField>" substring
+	Given "@BGM Research" was loaded
+	When user inputs "<PartValidNameField>" in algorithm search field
+	Then the field should be filtered
+
+Examples:
+	| PartValidNameField |
+	| Prob               |
+	| Regio              |
+
+Scenario Outline: 04 Search by upper case
+	Given "@BGM Research" was loaded
+	When user inputs "<ValidNameFieldUpperCase>" in algorithm search field
+	Then the field should be filtered
+
+Examples:
```

Comment:
You have only one variable value. It can be a usual scenario, not a scenario outline